src/legacy/core_plugins/kibana/public/home/components/tutorial/tutorial.js

```diff
@@ -349,14 +370,24 @@ class TutorialUi extends React.Component {
         </div>
       );
     }
+
+    let breadcrumbs;
+    if (!this.props.isK7Design) {
+      breadcrumbs = (
+        <EuiText>
+          <EuiLink href="#/home">{homeTitle}</EuiLink> /{' '}
+          <EuiLink href="#/home/tutorial_directory">{addDataTitle}</EuiLink> /{' '}
+          {this.state.tutorial ? this.state.tutorial.name : this.props.tutorialId}
+        </EuiText>
+      );
+    }
+
     return (
       <EuiPage className="homPage">
         <EuiPageBody>
 
-          <div>
-            <EuiLink href="#/home">Home</EuiLink> / <EuiLink href="#/home/tutorial_directory">Add Data</EuiLink>
-          </div>
-          <EuiSpacer size="s" />
```

Comment:
Do you want to keep this spacer? Maybe with something like:

```js
breadcrumbs = (
  <React.Fragment>
    <EuiText>
      <EuiLink href="#/home">{homeTitle}</EuiLink> /{' '}
      <EuiLink href="#/home/tutorial_directory">{addDataTitle}</EuiLink> /{' '}
      {this.state.tutorial ? this.state.tutorial.name : this.props.tutorialId}
    </EuiText>
    <EuiSpacer size="s" />
  </React.Fragment>
);
```

Reply:
I think EuiText gives enough space without the spacer? What do you think @cchaos?

Reply:
Looking into it further, I don't think `EuiText` is necessary here. All it's providing is extra line-height. I'd opt for removing that and keeping in the spacer.